kore/test/Test/Kore/ASTVerifier/DefinitionVerifier/PatternVerifier.hs

```diff
@@ -1237,19 +1228,6 @@ genericPatternInPatterns
                         ++ "' (<test data>)"
                     ]
             }
-        , TestPattern
```

Comment:
Why is this test removed?

Reply:
I removed this before I figured out the correct implementation for the verification, and it looked like it would never pass under the new rules. I added it back now.